CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.9.0
+
+* Added support for Python 3.13.
+* Fixed loading additional certificates on macOS Secure Transport.
```

Comment:
I don't think we're technically using the Secure Transport APIs

Reply:
Oh true, I'll fix this.